### PR TITLE
MCR-1950-Added_schemaOrg_support

### PR DESCRIPTION
**Pull request by paschty**

### CODE REVIEW

**Comment by paschty** [comment]
We also need the schema.org Classification can be created with: https://gist.github.com/paschty/79f909097c8e2404923fea73313952d1 and https://bib.schema.org/version/3.4/schema.rdf

**Comment by MyCoRe-Travis** [comment]
Something gone wrong with 59103afcb8c42dcc09c1855b4f74721e409e0506! 
 Please Check https://github.com/MyCoRe-Travis/MIR_test_artifacts/tree/1788-pull_request-316

**Comment by kkrebs** [comment]
please rebase and solve conflicts

File: mir-module/src/main/resources/xsl/mir2schemaOrg.xsl [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with MyCoRe.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                xmlns:mods="http://www.loc.gov/mods/v3"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+>
+
```
Comment (yagee-de) [inline mir-module/src/main/resources/xsl/mir2schemaOrg.xsl]: transitive include is missing

**Pull request merged**